src/blue/BlueSnippets.sol

```diff
@@ -193,102 +196,99 @@ contract BlueSnippets {
      */
     function supplyCollateral(MarketParams memory marketParams, uint256 amount, address user) external {
```

Comment:
I would even remove the possibility to supply/repay on behalf of someone

Assigning `onBehalf = msg.sender` is enough to me to point out the feature